NumberNinjaServer/routes/users.js

```diff
@@ -0,0 +1,103 @@
+var express = require('express');
+var router = express.Router();
+var User = require('../models/user');
+var jwt = require('jsonwebtoken');
+
+/* POST API to register user details to users collection of 
+ * MongoDB database and also check whether the user's email
+ * is registered before.
+ */
+router.post('/register',  function(req,res,next){
+  var user = new User({
+    email: req.body.Email,
+    username: req.body.UserName,
+    password: User.hashPassword(req.body.Password),
+    firstname: req.body.FirstName,
+    lastname: req.body.LastName,
+    role: null,
+    creation_dt: Date.now()
```

Comment:
Please follow camelCase format for variable use.

Reply:
Done.